apps/web/src/app/(site)/costs/client-page.tsx

```diff
@@ -0,0 +1,164 @@
+                        </Suspense>
+                    </ErrorBoundary>
+                    <ErrorBoundary fallback={EnergyCostsChangeLastThirtyDaysError}>
+                        <Suspense fallback={<Skeleton className="h-40 w-full" />}>
+                            <EnergyCostsChangeLastMonth userData={userData} energyData={energyDataRaw} />
+                        </Suspense>
+                    </ErrorBoundary>
+                    <ErrorBoundary fallback={EnergyCostsChangeLastSevenDaysNationalAverageError}>
+                        <Suspense fallback={<Skeleton className="h-40 w-full" />}>
+                            <EnergyCostsChangeLastWeekNationalAverage userData={userData} energyData={energyDataRaw} />
+                        </Suspense>
+                    </ErrorBoundary>
+                    <ErrorBoundary fallback={EnergyCostsChangeLastThirtyDaysNationalAverageError}>
+                        <Suspense fallback={<Skeleton className="h-40 w-full" />}>
+                            <EnergyCostsChangeLastMonthNationalAverage userData={userData} energyData={energyDataRaw} />
+                        </Suspense>
+                    </ErrorBoundary>
+                </div>
+            </section>
+
+            {/* Hochrechnungen */}
+            <section>
+                <h2 className="mb-4 font-bold text-xl">Hochrechnungen</h2>
+                <div className="grid grid-cols-1 gap-4 lg:grid-cols-3 md:grid-cols-2">
+                    <Suspense fallback={<Skeleton className="h-40 w-full" />}>
+                        <EnergyCostsProjectionDay userData={userData} energyData={energyDataRaw} />
+                    </Suspense>
+                    <Suspense fallback={<Skeleton className="h-40 w-full" />}>
+                        <EnergyCostsProjectionWeek userData={userData} energyData={energyDataRaw} />
+                    </Suspense>
+                    <Suspense fallback={<Skeleton className="h-40 w-full" />}>
+                        <EnergyCostsProjectionMonth userData={userData} energyData={energyDataRaw} />
+                    </Suspense>
+                    <Suspense fallback={<Skeleton className="h-40 w-full" />}>
+                        <EnergyCostsComparativeProjectionDay userData={userData} energyData={energyDataRaw} />
+                    </Suspense>
+                    <Suspense fallback={<Skeleton className="h-40 w-full" />}>
+                        <EnergyCostsComparativeProjectionWeek userData={userData} energyData={energyDataRaw} />
+                    </Suspense>
+                    <Suspense fallback={<Skeleton className="h-40 w-full" />}>
+                        <EnergyCostsComparativeProjectionMonth userData={userData} energyData={energyDataRaw} />
+                    </Suspense>
+                </div>
+            </section>
+
+            {/* Tägliche Energiekosten Bar Chart */}
+            <section>
```

Comment:
Würde das Chart noch in eine Card packen. So haben Malte und Daniel das auch gemacht